src/dev/jest/config.js

```diff
@@ -49,8 +49,6 @@ export default {
     '!packages/kbn-ui-framework/src/services/**/*/index.js',
     'src/legacy/core_plugins/**/*.{js,jsx,ts,tsx}',
     '!src/legacy/core_plugins/**/{__test__,__snapshots__}/**/*',
-    'src/legacy/ui/public/{agg_types,vis}/**/*.{ts,tsx}',
-    '!src/legacy/ui/public/{agg_types,vis}/**/*.d.ts',
```

Comment:
There are no necessity to collect coverage from these folders (`agg_types` doesn't exist anymore, `vis` contains only legacy mocka tests since the most of code was moved)

Reply:
👍